pkg/labels/labels.go

```diff
@@ -511,9 +511,10 @@ func GetCiliumKeyFrom(extKey string) string {
 func GetExtendedKeyFrom(str string) string {
 	src, next := parseSource(str, ':')
 	if src == "" {
-		src = LabelSourceAny
+		//src = LabelSourceAny
```

Comment:
I'm a bit confused by this - wouldn't the `foo=bar` example in the doc comment now return `foo=bar` unchanged? 

Or if that's intended, then nit: leftover comment.